src/objects.js

```diff
@@ -1,41 +1,49 @@
 const createPerson = (name, age) => {
-  // your code here
+  return {name, age};
 };
 
 const getName = object => {
-  // your code here
+  return object.name;
 };
 
 const getProperty = (property, object) => {
-  // your code here
+  return object[property];
 };
 
 const hasProperty = (property, object) => {
-  // your code here
+  if (object[property]) return true;
+  return false;
 };
 
 const isOver65 = person => {
-  // your code here
+  if (person.age > 65) return true;
+  return false;
 };
 
 const getAges = people => {
-  // your code here
+  return people.map(newArray => newArray.age);
```

Comment:
This is correct and you can't get it shorter. The only suggestion I have is that I would rename `newArray` to be `person` as it represents a single value from the `people` array.